examples/embark.json

```diff
@@ -0,0 +1,8 @@
+{
+  "page": "#1e1c31",
+  "text": "#cbe3e7",
+  "background": "#3E3859",
+  "highlight": "#a1efd3",
```

Comment:
Looking much better! Sorry I realize now how the highlight and highlight active highlights work. The active should be the lighter color #FFE6B3 and the non active the darker #F2B482.

Reply:
Highlight is set to `#a1efd3` instead of `#F2B482`.

Reply:
Fixed.